.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+ - [ ] fixes #
+ - [ ] includes new source code with proper [APACHE license header](https://github.com/ipa320/setup/tree/master/templates)
```

Comment:
I would do something like

`includes new source code --> this must have the proper ...`

Otherwise, I could add code without the header and just don't have to check the box ;-)